components/products/products.controller.js

```diff
@@ -0,0 +1,18 @@
+const productsController = function ($scope, myService, $location) {
+  this.arrayProducts = [];
+
+  this.getAllProducts = async () => {
+    const prods = await myService.getProducts();
```

Comment:
naming issue: "prods" should be "response" I guess (because I see you wrote prods.products later on)